lib/pavilion/plugins/commands/result.py

```diff
@@ -191,7 +205,7 @@ def _get_tests(self, pav_cfg, tests_arg):
         return map(int, test_list)
 
     def update_results(self, pav_cfg: dict, tests: List[TestRun],
-                       log_file: IO[str]) -> bool:
+                       log_file: IO[str], save=False) -> bool:
```

Comment:
Add the type hint for save, `save: bool = False`, and add it to the params in the doc string.